agent/taskresource/asmauth/asmauth_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
//+build unit

Reply:
added.

Reply:
Did you miss this, I didn't see this was added?

Reply:
yea missed it, added now.